src/lambdaconnect_sync/db_interface.cljc

```diff
@@ -0,0 +1,61 @@
+(ns lambdaconnect-sync.db-interface)
```

Comment:
Komentarz ogólny do nazewnictwa funkcji:
Część jest postaci `Y-by-X` lub `Y-for-X`, ja z kolei pisałbym `X->Y` albo `X-to-Y`. Nie chcę narzucać konwencji, ale może warto to ujednolicić?

Reply:
Tylko ze ten protokół potencjalnie musi być implementowany w javascripcie...?